src/examples/DataCenter/Input/paraCfgCurrent.txt

```diff
@@ -0,0 +1,20 @@
+ParameterName	ParameterValue
+environmentFileName	DataCenter.xml
+generalizedScenario	1
+indoorSwitch	1
+mobilitySwitch	0
+mobilityType	1
+numberOfNodes	2
+numberOfTimeDivisions	15
```

Comment:
15 lines in the position files means 14 time divisions

Reply:
These values are useless for a stationary scenario.

Reply:
but you can choose one of the locations in nodes file. I mean these 14 locations are the ones that they considered in their measurement.

Reply:
it doesn't have to directly translate into mobility. if they did 14 (or 15?) measurements then they can be considered as mobile nodes over an unspecified amount of time (e.g., 1s), especially if you wrote all of them down in the NodePosition files.
I would set
mobilitySwitch	1
mobilityType	2
numberOfNodes	2
numberOfTimeDivisions	15

Also, since the last position is discarded, please copy paste the last position in both NodePosition files so that you will have 16 lines and, thus, 15 numberOfTimeDivisions

Reply:
I already had the same thought but we cannot considered mobility because the locations they considered are on RackTop
So let it be static at this moment.

Reply:
Again, it may not be considered as mobility, just think of it as showing N measurements at once. Remember that the examples also work as tests, so having more outputs to test can actually be beneficial. Also, I'm not sure if these settings will take the first line of the NodePosition files or just consider a random position.

Reply:
Can you please elaborate. if you consider position values in NodePositions files, how is it not considered mobility? Truly speaking I am confused with this statement. 

Right now with the current setting, it is taking Tx and Rx position from Nodes.dat file. 

I don't have any problem if we can give all 15 position at the same time but nodes should be static. Please let me know how to do it.

Reply:
Well, having multiple positions per node is normally interpreted as "mobility", that's why it's called so.
The only thing that changes between mobility and multiple positions for a node is in fact a velocity vector that includes doppler shift into the channel. Notably, the current implementation ignores doppler shift and thus "mobility" is actually used inappropriately here..
Bottom line: the channel is computed as if they were 15 static node positions, so in general our simulator supports mobility as multiple independent node positions.
It would not be incorrect to use the node position file to list a number of independent positions, it is actually incorrect to call it mobility. Why? Well, that's how it works for the moment :)

Reply:
But I am still not convinced to consider mobility settings on the Racks even if this is not related to mobility.  Please consider the original setting as I considered. please let me know totalTimeDuration?

Reply:
ok, do whatever you prefer. it doesn't make sense to set the duration to 14 seconds, though, if you have a static scenario. I would set it either to 0 or to 1. Does it change anything in practice? No. But it's going to be public so I'd like to have at least some clean examples.